.github/workflows/dev-cluster-check.yml

```
@@ -0,0 +1,40 @@
+name: Check development cluster
+on:
+  schedule:
+    - cron: '* 0/2 * * *'
```

thinking if the expression should be rather like:
```suggestion
    - cron: '0 0/2 * * *'
```